src/index.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { FC, HTMLAttributes, useState, useCallback } from 'react';
 import styled from 'styled-components';
-import TextareaAutosize from 'react-autosize-textarea';
 import useDimensions from 'react-use-dimensions';
 import { ConvertCard } from './styled/ConvertCard';
 import { IconX } from './IconX';
 import { IconCopy } from './IconCopy';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import { InMenuOptionStuff } from './InMenuOptionStuff';
 import { OutMenuOptionStuff } from './OutMenuOptionStuff';
-import { IInOption, InOptions, IOutOption, OutOptions } from './types';
```

Comment:
Hey, I think you should not have to delete the imports here. They are later exported:

```tsx
export { IInOption, IOutOption, InOptions, OutOptions };
```

And are necessary to have types for the users of this package.